tests/test_data/test_projects/common_scargo_project_stm32/src/test_lib.h

```diff
@@ -0,0 +1,18 @@
+//
```

Comment:
Can we modify those test projects? I heard once on our daily that those projects are used to test backward compatibility. @aaksenczuk do you know anything about this?

Reply:
As discussed at the meeting this change is acceptable